### DISCUSSION TITLE
UI not upscaling at 125%

### DISCUSSION THREAD

**Discussion by Carel3DS** [opentoonz/opentoonz#6365] [Q&A]
**Describe the bug**
Having the UI scale set at 125% in Windows, OpenToonz UI does not rescale to the appropiate density

**To Reproduce**
List steps to reproduce the behavior.
1. Set Screen Scale at 125% by going Settings>System>Screen
2. Start OpenToonz

**Expected behavior**
OpenToonz UI have to be bigger

**Screenshots**
![Captura de pantalla (145)](https://user-images.githubusercontent.com/52781415/125511344-4d249afc-b87e-4f81-aaaf-bef4163691bb.png)

**Device Information:**
 - OS: Windows 10 21H1 (19043.1083)
 - OpenToonz Version: 1.5
 - GPU: NVIDIA GTX 1650
 - GPU Driver Version: 471.11

**Comment by RodneyBaker** [comment]
Hi @Carel3DS 
I don't have a solution to the problem but want to let you know your issue has been read.
I too have a similar issue on my laptop where Windows recommended setting of 125% doesn't allow all of Opentoonz (or Tahaoma2D) to appear on screen.  Interestingly I mostly experience the problem with T2D and not OT.

It isn't clear to me why I have the problem sometimes and not others.

The best I can suggest for the present is to check your graphics drivers and make sure they are either current or (if the most current is problematic) is the best for your system.  In the latter case you would have to roll back to an earlier driver.
